db/models.py

```diff
@@ -1 +1,57 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
+
+    def __str__(self) -> str:
+        return (f"name: {self.name}\n"
+                f"description: {self.description}")
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(
+        "kind of bonus players can get from it",
+        max_length=255)
+    race = models.ForeignKey(Race, on_delete=models.CASCADE)
+
+    def __str__(self) -> str:
+        return (f"name: {self.name}\n"
+                f"bonus: {self.bonus}\n"
+                f"race: {self.race}")
+
+
+class Guild(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(null=True)
+
+    def __str__(self) -> str:
+        return (f"name: {self.name}\n"
+                f"description: {self.description}")
+
+
+class Player(models.Model):
+    nickname = models.CharField(max_length=255, unique=True)
+    email = models.EmailField(max_length=255)
+    bio = models.CharField(
+        "It stores a short description "
+        "provided by a user about himself/herself",
+        max_length=255
+    )
+    race = models.ForeignKey(Race, on_delete=models.CASCADE)
+    guild = models.ForeignKey(
+        Guild,
+        on_delete=models.SET_NULL,
+        null=True,
+        default=None)
```

Comment:
Do not forget to place the closing parenthesis `)` on a new line in case you start a new line after opening the parenthesis `(`. Add this to your code.